packages/components/src/InputTime/InputTime.tsx

```diff
@@ -38,50 +45,16 @@ export function InputTime({
 
   function handleBlur() {
     params.onBlur?.();
+    console.log("handleBlur");
 
     // Time inputs doesn't clear the typed value when it's invalid. This should
     // force it to reset the input value when the typed value is invalid.
     if (ref.current) {
+      // when would this ever be invalid? we don't allow any
+      // of the attributes that would make it invalid
       if (!ref.current.checkValidity()) {
```

Comment:
I'm unclear how we'd ever hit this block. I must be missing something because it seems to me that we'd always be comparing a timestamp to some string value.

<img width="288" alt="image" src="https://github.com/user-attachments/assets/accc5b80-6958-48ea-859f-811f2c2108ad" />
<img width="814" alt="image" src="https://github.com/user-attachments/assets/9c2ba8b9-9193-485a-8dc5-0f2e9b8a6842" />
https://developer.mozilla.org/en-US/docs/Web/HTML/Constraint_validation

doesn't really matter in the context of what this PR is meant to achieve though.